src/zenml/model/model.py

```diff
@@ -73,12 +73,12 @@ class Model(BaseModel):
     tags: Optional[List[str]] = None
     version: Optional[Union[ModelStages, int, str]] = None
     save_models_to_registry: bool = True
+    id_: Optional[UUID] = None
```

Comment:
The changes look good! However, when I first started the review, the naming tricked me a bit. Since this is the `Model` class, I expected the id to be associated with a `ModelResponse` rather than a `ModelVersionResponse`. Since we already have the `model_id`, if it is possible, I would recommend changing this to a `model_version_id`.

Reply:
Since it is still meant to be local only, I don't care much. This class specifically was renamed into `Model` in the course of concepts flattening some time back. I'll rename it.